lib/matplotlib/backends/_backend_pdf_ps.py

```diff
@@ -16,11 +16,43 @@ def _cached_get_afm_from_fname(fname):
         return AFM(fh)
 
 
+class CharacterTracker:
+    def __init__(self):
```

Comment:
I would love to see a bit more documentation on the class itself and its public methods.

Reply:
done